src/utils/index.ts

```diff
@@ -695,35 +695,29 @@ export function claimToMeshClaim(claim: Claim, context: Context): MeshClaim {
 /**
  * @hidden
  */
-export function stringToClaimType(claim: string): ClaimType {
-  if (claim === 'Accredited') {
-    return ClaimType.Accredited;
-  }
-  if (claim === 'Affiliate') {
-    return ClaimType.Affiliate;
-  }
-  if (claim === 'BuyLockup') {
-    return ClaimType.BuyLockup;
-  }
-  if (claim === 'SellLockup') {
-    return ClaimType.SellLockup;
-  }
-  if (claim === 'CustomerDueDiligence') {
-    return ClaimType.CustomerDueDiligence;
-  }
-  if (claim === 'KnowYourCustomer') {
-    return ClaimType.KnowYourCustomer;
-  }
-  if (claim === 'Jurisdiction') {
-    return ClaimType.Jurisdiction;
-  }
-  if (claim === 'Whitelisted') {
-    return ClaimType.Whitelisted;
-  }
-  if (claim === 'Blacklisted') {
-    return ClaimType.Blacklisted;
-  }
-  return ClaimType.NoData;
+export function createClaim(
```

Comment:
```
export function createClaim(
  claimType: string,
  jurisdiction?: string | null,
  scope?: string | null
): Claim {
  const type = claimType as ClaimType;
  if (type === ClaimType.Jurisdiction) {
    return {
      type,
      name: jurisdiction as string,
      scope: scope as string,
    };
  } else if (type !== ClaimType.NoData && type !== ClaimType.CustomerDueDiligence) {
    return {
      type,
      scope: scope as string,
    };
  }

  return { type };
}
```

Replace it with this

For this to work though, the `Claim` type in types/index has to be replaced with 
```
export type Claim =
  | { type: ClaimType.Jurisdiction; name: string; scope: string }
  | { type: Exclude<ClaimType, ClaimType.NoData | ClaimType.Jurisdiction>; scope: string }
  | { type: ClaimType.NoData | ClaimType.CustomerDueDiligence };
```